src/compiler/utilities.ts

```diff
@@ -328,30 +328,29 @@ namespace ts {
             return getSourceTextOfNodeFromSourceFile(sourceFile, node);
         }
 
+        const escapeText = getEmitFlags(node) & EmitFlags.NoAsciiEscaping ?

```

Comment:
Consider moving the arrows out of the function as they don't close over any local state. Will reduce allocations for function objects.